src/refract/element.es6

```diff
@@ -0,0 +1,92 @@
+/*
+ * Base elements for working with refract shorthand, which is composed of
+ * tuples with the following values:
+ *
+ * [name, metadata, attributes, contents]
+ *
+ * For example:
+ *
+ * ['resource', {'title': 'Question'}, {'href': '/questions'},
+ *   ['transition', {}, {}, ...],
+ *   ...
+ * ]
+ *
+ * https://github.com/refractproject/refract-spec
+ */
+const rawElement = Symbol('element');
+
+export default class RefractElement {
```

Comment:
@kylef at the moment it doesn't seem to support metadata, custom element types or reading from compact refract. The first two are necessary for my use case. @smizell, should I be updating minim or do you have any plans around those features at the moment?